### DISCUSSION TITLE
Request new command: alr mrproper

### DISCUSSION THREAD

**Discussion by pmunts** [alire-project/alire#1307] [Ideas]
`alr mrproper` would be exactly equivalent to `rm -rf alire bin config obj share`, and removes everything that is or can be regenerated automatically.  It is also equivalent to `make mrproper` and
`make distclean` as are commonly used in `make` projects.

What remains after `alr mrproper` is the minimal portion of an Alire  project that is suitable for checking into a source code repository.

Adding a new option like `--mrproper` or `--pristine` or even `--all` to `alr clean` would be equivalent.

**Comment by Fabien-Chouteau** [comment]
I think a `--distclean` option for `alr clean` is the best way. We don't need a new command for that.

**Comment by jquorning** [comment]
note: In **brew** package manager for *Ruby*[1] we use `doctor`[2] for cleaning up the package library and setup.

[1]: https://brew.sh/
[2]: https://docs.brew.sh/Manpage#doctor-dr---list-checks---audit-debug-diagnostic_check-
